tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -31,11 +31,15 @@ EOF
 		curl -fsSL https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key |
 			gpg --dearmour -o $rootfs_dir/etc/apt/trusted.gpg.d/intel-sgx-deb.gpg
 		sed -i -e "s/bootstrap=Ubuntu/bootstrap=Ubuntu intel-sgx/" $multistrap_conf
+		SUITE=$UBUNTU_CODENAME
+		# Intel does not release sgx stuff for non-LTS, thus if using oracular (24.10),
+		# we need to enforce getting libtdx-attest from noble.
+		[ "$SUITE" = "oracular" ] && SUITE="noble"
```

Comment:
We are using jammy, not noble though?

Reply:
Sorry, I see we bumped to oracular later on, maybe we could squash this commit into the `latest`->`22.04` so the commit trail makes more sense?